samples/local_attestation/common/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# Use the edger8r to generate C bindings from the EDL file.
+add_custom_command(OUTPUT localattestation_t.h localattestation_t.c
+  COMMAND openenclave::oeedger8r --trusted ${CMAKE_SOURCE_DIR}/localattestation.edl)
+
+# Create a library common to each of our two enclaves.
+add_library(common STATIC attestation.cpp crypto.cpp dispatcher.cpp ${CMAKE_CURRENT_BINARY_DIR}/localattestation_t.c)
+target_compile_definitions(common PUBLIC OE_API_VERSION=2)
+target_link_libraries(common PUBLIC
+  # `liboecore`, a dependency of `liboeenclave`, requires the ecalls
```

Comment:
@andschwa I realize this is carried over from the already merged data_sealing sample, but we might want to clean this comment up as part of the doc-a-thon; it reads like it was intended for a maintainer audience (such as the vague suggestion for the alternative.)

Reply:
Yeah, probably!